ui/src/app/pages/login/account-creation/account-creation-institution.tsx

```diff
@@ -29,8 +29,7 @@ import {Dropdown} from 'primereact/dropdown';
 const styles = reactStyles({
   ...commonStyles,
   wideInputSize: {
-    width: '50%',
-    minWidth: '600px'
+    width: 600
```

Comment:
Please add a comment with a short description of the reasoning here and a cross-link to the Jira issue. It's a nuanced enough fix that it warrants some breadcrumbs.

Reply:
Got it, will do.

Reply:
The flex issue here is counter-intuitive. If you set the min-width to 600px on the first child of the top level flex container (the first column) it should resolve the issue. 

The reason the min-width needs to be set is because the min-width of a flex item defaults to 'auto'. If the text is not wrapped, as is the case with the selected item in the drop down, the browser will compute the min width for the flex item as 'auto' essentially sizing the item to the content. Setting a fixed min-width will correct for this.

You can read more here - although it is verbose
https://www.w3.org/TR/css-flexbox-1/#min-size-auto

Reply:
Nice catch – and with a w3c reference to boot!

My assertion about the fix being "obscure" may still stand, but I was wrong about it being more complicated than the workaround in this PR – it actually feels much simpler that way.